arrangor/src/main/kotlin/no/nav/amt/tiltak/ansatt/ArrangorAnsattServiceImpl.kt

```diff
@@ -84,6 +90,37 @@ class ArrangorAnsattServiceImpl(
 		arrangorAnsattRepository.setVelykketInnlogging(ansattId)
 	}
 
+	override fun migrerAlle() {
```

Comment:
Vi skriver alle arrangør-ansatte til topicen `amt.arrangor-ansatt-v1`. Er det ikke enklere om du leser fra den? Da får du en løsning som funker også når det ikke lenger er amt-tiltak som er master for arrangør (som jeg jobber med nå) :)

Reply:
Kom ikke på at den fantes, kan godt bruke den 👍

Reply:
Alle ansatte blir lastet på nytt der hver natt, så tror du skal kunne stole på at den er forholdsvis komplett :D Vi har en topic for deltakere også, men nav-ansatte mangler vel kanskje, så for nav-ansatte må du nok gjøre sånn du har gjort her da.

Reply:
hmm ja det kan hende ja..